src/java/voldemort/server/VoldemortConfig.java

```diff
@@ -406,6 +407,7 @@ public VoldemortConfig(Props props) {
 
         this.modifiedProtocol = props.getString("readonly.modify.protocol", null);
         this.modifiedPort = props.getInt("readonly.modify.port", -1);
+        this.bouncyCastleEnabled = props.getBoolean("readonly.bouncy.castle.enabled", false);
```

Comment:
Probably not in the config class, but some place common. Leaving it in the HDFSFetcher alters the behavior of the program mid way and it will have inconsistent behavior for some paths. It need to be done in the startup so the behavior is consistent, let me look for a place to put this in.